nf_core/pipeline-template/tests/default.main.nf.test

```diff
@@ -0,0 +1,22 @@
+nextflow_pipeline {
+
+    name "Test pipeline with default settings"
+    script "../main.nf"
+    // tag  {{ name }}
+    tag "PIPELINE"
+
+    test("Default profile test") {
+
+        when {
+            params {
+                outdir = "$outputDir"
+            }
+        }
+
+        then {
+            assertAll(
+                { assert workflow.success }
```

Comment:
We should add "TODO nf-core" comments and link to documentation on how to assert different files, are more tests with other profiles, etc.